### DISCUSSION TITLE
How to calibrate

### DISCUSSION THREAD

**Discussion by Erwinsmith101** [ESPresense/ESPresense#353] [Q&A]
Can somone help and explain to me how to calibrate the distance…

**Comment by Delfairen** [comment]
I'd also like to know this and also how to calibrate for different devices as they will show different distances even when right next to each other.

**Comment by nightshade00013** [comment]
https://github.com/ESPresense/ESPresense/issues/324#issuecomment-1064004098 has some information about calibrating things.  At this point there is not an individual adjustment but if you are using HomeAssistant with the app on a device you can make adjustments in the BLE sensor section.
